### ISSUE TITLE
Frequency about LiDAR point cloud data

### ISSUE THREAD

**Issue by hehefan** [argoai/argoverse-api#70]
Hi,

I found 13,395 pcs in training set, 4,816 pcs in val set and  4,189 pcs in test set. However, I did not find the frequency information about these pc records. What is the time interval between two successive pcs?

**Comment by James-Hays** [comment]
The lidar "frames" are captured at 10hz or 100ms apart.

The notion of lidar "frames" is a bit artificial, though, because they are continually acquiring returns (so two returns in the same frame could be 99ms apart, and two returns in adjacent frames could be 1ms apart). 

To complicate things further, in Argoverse, there are two lidars operating out of phase. So a single location in the world is swept by one of the lidars every 50ms. 

Also see #20 and #37

**Comment by hehefan** [comment]
Many thanks! @James-Hays
